src/DynamoCoreWpf/Views/GuidedTour/GuideBackground.xaml.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Dynamo.Wpf.ViewModels.GuidedTour;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+
+namespace Dynamo.Wpf.Views.GuidedTour
+{
+    /// <summary>
+    /// Interaction logic for GuideBackground.xaml
+    /// </summary>
+    public partial class GuideBackground : UserControl
+    {
+        public GuideBackground()
+        {
+            InitializeComponent();
+            DataContext = this;
+        }
+
+        /// <summary>
+        /// Width of the background
+        /// </summary>
+        public double CanvasWidth
+        {
+            get { return (double)GetValue(CanvasWidthProperty); }
+            set { SetValue(CanvasWidthProperty, value); }
+        }
+
+        /// <summary>
+        /// Height of the background
+        /// </summary>
+        public double CanvasHeight
+        {
+            get { return (double)GetValue(CanvasHeightProperty); }
+            set { SetValue(CanvasHeightProperty, value); }
+        }
+
+        /// <summary>
+        /// Rect with the sieze of the Dynamo Window 
```

Comment:
size misspelled?

Reply:
Correct, will update this on next commit